File: easy/5_check_prime/check_prime_test.go
package main

import "testing"

func TestPrime(t *testing.T) {
	if isPrime(1) {
		t.Errorf("isPrime(1) = true, false expected")
	}
	if !isPrime(17) {
		t.Errorf("isPrime(17) = false, true expected")
	}
	if !isPrime(2) {
		t.Errorf("isPrime(2) = false, true expected")
	}
	if !isPrime(3) {
		t.Errorf("isPrime(3) = false, true expected")
	}
}

File: easy/5_check_prime/check_prime_test.py
import unittest
from check_prime import *

class TestPrime(unittest.TestCase):
    def testPrime(self):
        self.assertEqual(is_prime(1), False)
        self.assertEqual(is_prime(17), True)
        self.assertEqual(is_prime(2), True)
        self.assertEqual(is_prime(3), True)